packages/replay/src/main.ts

```diff
@@ -547,7 +548,9 @@ async function doViewRecording(
     }
   }
   const dispatch = server != "wss://dispatch.replay.io" ? `&dispatch=${server}` : "";
-  spawn(openExecutable(), [`https://app.replay.io?id=${recordingId}${dispatch}`]);
+  spawn(openExecutable(), [
+    `${devtoolsServer ?? "https://app.replay.io"}?id=${recordingId}${dispatch}`,
```

Comment:
nit: I think it would be slightly nicer to assign this to its own variable like the line above.